string[] ids     # ids to classify

---

string[] ids     # ids that where classified

CategoricalDistribution[] posteriors   # Posterior probability distribution per id

# The following is just for convenience, and can be derived from the posteriors:
string[] expected_values                # Expected values based on posteriors, per id (empty is unknown)
float32[] expected_value_probabilities  # Expected value probabilities, per id

string error_msg

